dubbo-config/dubbo-config-api/src/test/java/org/apache/dubbo/config/bootstrap/DubboBootstrapTest.java

```diff
@@ -1,406 +0,0 @@
-/*
```

Comment:
Why were these unit tests removed?

Reply:
They are either using the rest protocol or using the rest protocol-specific userservice definition

Reply:
Please remove only the rest-related parts, do not delete the entire unit test class.